lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -176,6 +176,11 @@
     .forum-nav-thread-labels {
         margin: 5px 0 0;
     }
+
+    .thread-preview-body {
+        color: $forum-color-response-count;
+        font-size: $forum-small-font-size;
```

Comment:
DiscussionUtil.abbreviateString already has an ellipse appended to the truncated text, with translation, too.